src/styles/blocks/main-container.scss

```diff
@@ -0,0 +1,23 @@
+.main {
+  margin: 0;
+  display: grid;
+  grid-template-columns: repeat($column-count, 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+  padding-top: 50px;
+  padding-bottom: 50px;
+
+  @media (min-width: 488px) {
+    $column-count: 2;
+  }
+
+  @media (min-width: 768px) {
+    $column-count: 3;
+  }
+
+  @media (min-width: 1024px) {
+    $column-count : 4;
```

Comment:
In this task, it's not necessary, but sass variables are not dynamic and the column-count will not change
https://stackoverflow.com/questions/9122195/using-sass-variables-with-css3-media-queries

Reply:
Done)